docs/datascience/jupyter-notebooks.md

```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
 
 ![Debug cell button](images/jupyter/debug-cell.png)
 
+### Search through notebook
+
+You can search (Ctrl / Cmd + F) through an entire notebook or you can search parts of your notebook by filtering your search options. When searching through your notebook, click on the Filter option (funnel icon) to search across:
+- Markdown cell input (`Markdown Source`)
+- Markdown cell output (`Rendered Markdown`)
+- Code cell input (`Code Cell Source`)
+- Code cell output (`Cell Output`)
+
+Notebook searches are filtered cell inputs only by default.
+
+![Search options](images/jupyter/search-filter.png)
+
 ## Connect to a remote Jupyter server
 
 You can offload intensive computation in a Jupyter Notebook to other computers by connecting to a remote Jupyter server. Once connected, code cells run on the remote server rather than the local computer.
 
 To connect to a remote Jupyter server:
 
-1. Select the **Jupyter Server: local** button in the global Status bar or run the **Jupyter: Specify local or remote Jupyter server for connections** command from the Command Palette (`kb(workbench.action.showCommands)`).
+1. Open the Kernel Picker button on the top right-hand side of the notebook (or run the `Notebook: Select Notebook Kernel` command from the Command Palette).
+
+   ![Notebook kernel picker](images/jupyter/notebook-kernel-picker.png)
+
+2. Select the **Existing Jupyter Server** option to connect to an existing Jupyter server.
 
-   ![Specify remote Jupyter server](images/jupyter/specify-remote.png)
+   ![Select existing Jupyter server](images/jupyter/select-existing-server.png)
 
-1. When prompted to **Pick how to connect to Jupyter**, select **Existing: Specify the URI of an existing server**.
+3. To connect to an existing server for the first time, select **Enter the URL of the running Jupyter server**.
 
-   ![Choose to connect to an existing server](images/jupyter/connect-to-existing.png)
+   ![Choose to connect to an existing server](images/jupyter/select-enter-server-url.png)
 
-1. When prompted to **Enter the URI of a Jupyter server**, provide the server's URI (hostname) with the authentication token included with a `?token=` URL parameter. (If you start the server in the VS Code terminal with an authentication token enabled, the URL with the token typically appears in the terminal output from where you can copy it.) Alternatively, you can specify a username and password after providing the URI.
+4. When prompted to **Enter the URI of a Jupyter server**, provide the server's URI (hostname) with the authentication token included with a `?token=` URL parameter. (If you start the server in the VS Code terminal with an authentication token enabled, the URL with the token typically appears in the terminal output from where you can copy it.) Alternatively, you can specify a username and password after providing the URI.
```

Comment:
Should this be "Enter the URL of the running Jupyter server" to make the prompt?

Reply:
👍🏻